main.cpp

```diff
@@ -95,6 +96,12 @@ int main(int argc, char** argv)
             untrusted_dir = get_dir(i++);
         else if (a == "--experimental")
             ;
+        else if (a.rfind("-D", 0) == 0)
+        {
+            std::string define = a.substr(2);
+            if (!define.empty())
```

Comment:
print usage if define is empty.

Reply:
print the same error message as gcc when the define the empty.